hc/api/tests/test_create_check.py

```diff
@@ -49,15 +49,17 @@ def test_it_accepts_api_key_in_header(self):
         payload = json.dumps({"name": "Foo"})
 
         ### Make the post request and get the response
-        r = {'status_code': 201} ### This is just a placeholder variable
-
-        self.assertEqual(r['status_code'], 201)
+        r = self.post({
```

Comment:
This is great work, but consider using more descriptive variable names as opposed to one letter variables. This would for instance be _**response**_

Reply:
I think this will add to PEP8 consistency